kahuna/public/js/components/gr-export-original-image/gr-export-original-image.html

```diff
@@ -0,0 +1,14 @@
+<a class="drop-menu inner-clickable clickable side-padded button-right-side" ng:click="showUserActions = !showUserActions">
+    <span>▾</span>
+</a>
+
+<div ng:if="showUserActions"
+     class="drop-menu__items drop-menu__items--nopad"
+     ng:class="{ 'drop-menu__items--action' : ctrl.cropping }"
+     ng:click="ctrl.callCrop()">
+    <ul>
```

Comment:
In theory, this could be a useful component, so I'd be tempted to keep the flexibility...